docs/pages/management/admin/self-signed-certs.mdx

```diff
@@ -0,0 +1,224 @@
+
+  Configuring your cluster to trust self-signed certificates
+  makes it easier for attackers to intercept communications
+  between the Proxy Service and clients, since there is no
+  way to verify the authenticity of the certificates.
+  It is therefore important to properly configure certificates
+  when using Teleport in a production environment.
+</Notice>
+
+## Prerequisites
+
+<Tabs>
+<TabItem scope={["oss"]} label="Open Source">
+
+- A running Teleport cluster. For details on how to set this up, see one of our
+  [Getting Started](/docs/getting-started) guides (skip TLS certificate setup).
+
+- A Teleport Proxy Service which does not have certificates or ACME automatic certificates configured.
+For example, this Teleport Proxy Service configuration would use self-signed certs:
+
+  ```yaml
+  proxy_service:
+    enabled: "yes"
+    # TLS certificate for the HTTPS connection.
+    https_keypairs: []
+    # Get an automatic certificate from letsencrypt.org using ACME.
+    acme: {}
+  ```
+
+- The `tctl` admin tool and `tsh` client tool version >= (=teleport.version=).
+
+  ```code
+  $ tctl version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+
+  $ tsh version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  ```
+
+  See [Installation](/docs/installation.mdx) for details.
+
+</TabItem>
+<TabItem
+  scope={["enterprise"]} label="Enterprise">
+
+- A running Teleport cluster. For details on how to set this up, see our Enterprise
+  [Getting Started](/docs/enterprise/getting-started) guide.
+
+- A Teleport Proxy Service which does not have certificates or ACME automatic certificates configured.
+For example, this Teleport Proxy Service configuration would use self-signed certs:
+
+  ```yaml
+  proxy_service:
+    enabled: "yes"
+    # TLS certificate for the HTTPS connection.
+    https_keypairs: []
+    # Get an automatic certificate from letsencrypt.org using ACME.
+    acme: {}
+  ```
+
+- The `tctl` admin tool and `tsh` client tool version >= (=teleport.version=),
+  which you can download by visiting the
+  [customer portal](https://dashboard.gravitational.com/web/login).
+
+  ```code
+  $ tctl version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  
+  $ tsh version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  ```
+
+</TabItem>
+</Tabs>
+
+## How to use self-signed certs with Teleport binaries
+
+### `teleport`
+
+When running a Teleport service with `teleport`, if the service you are starting is configured to point to
+the Teleport Proxy Service endpoint and the Proxy Service is using self-signed certificates, then `teleport` will need
+to be run with the `--insecure` flag to disable verification of the
+Proxy Service TLS certificate. This is the case when:
+- The Teleport config file `auth_servers` setting is set to the Proxy Service endpoint:
+  - `auth_servers: ["tele.example.com:443"]` or
+  - `auth_servers: ["tele.example.com:3080"]`
```

Comment:
We added config v3 which has either `auth_server` or `proxy_server` 

https://github.com/gravitational/teleport/blob/bd0ae3819b27e107153398de926392c96f8088ae/docs/pages/reference/config.mdx?plain=1#L142-L158

Can you update this to match please?

Note, we didn't backport this, so when backporting for v10 you should still mention `auth_servers`.

Reply:
Sure. On that note, are we planning to update the `teleport start` cli to have separate `--proxy-server` and `--auth-server`?

Reply:
cc @zmb3